### DISCUSSION TITLE
关于显存占用

### DISCUSSION THREAD

**Discussion by Ethereal-sakura** [ymcui/Chinese-Mixtral#10] [General]
作者您好，我想请教一下128K，64K上下文，需要的显存分别是多少？

**Comment by ymcui** [comment]
信息不足，无法给出准确数据。什么工具，什么精度？你参考以下数据吧。

llama.cpp下Q6_K（GGUF格式），35.74 GiB（磁盘占用大小）：
- 4K长度占用37118MB
- 16K长度占用38654MB
- 64K长度占用44797MB
- 128K长度占用52606MB
